tests/stress_tests/.automation/memcheck_tests/precommit_configs/desktop_references_config.xml

```diff
@@ -3,53 +3,53 @@
     <models>
         <!--Models with FP32 precision-->
         <model path="public/mobilenet-ssd/FP32/mobilenet-ssd.xml" precision="FP32" test="create_exenetwork" device="CPU" vmsize="740214" vmpeak="805110" vmrss="129308" vmhwm="129308" />
-        <model path="public/mobilenet-ssd/FP32/mobilenet-ssd.xml" precision="FP32" test="create_exenetwork" device="GPU" vmsize="2709506" vmpeak="2794703" vmrss="1342104" vmhwm="1342104" />
+        <model path="public/mobilenet-ssd/FP32/mobilenet-ssd.xml" precision="FP32" test="create_exenetwork" device="GPU" vmsize="1935798" vmpeak="1958798" vmrss="467729" vmhwm="796421" /> # values from {"commit_id": "2947789b3b18a724096abbd9a5c535ae3128ce05", "commit_date": "2021-07-12 23:30"} and *= 1.3
```

Comment:
@ValentinDymchishin could you comment how results were collected? It should be collected from CI run (e.g. commit validation on master branch)